mailpoet/assets/js/src/wizard/steps/pitch_mss_step/first_part.tsx

```diff
@@ -0,0 +1,81 @@
+import { useHistory, useParams } from 'react-router-dom';
+import { external, Icon } from '@wordpress/icons';
+import { Heading } from '../../../common/typography/heading/heading';
+import { MailPoet } from '../../../mailpoet';
+import { List } from '../../../common/typography/list/list';
+import { Button } from '../../../common';
+import { OwnEmailServiceNote } from './own_email_service_note';
+
+type MSSStepFirstPartPropType = {
+  subscribersCount: number;
+  finishWizard: (redirect_url?: string) => void;
+};
+
+function MSSStepFirstPart({
+  subscribersCount,
+  finishWizard,
+}: MSSStepFirstPartPropType): JSX.Element {
+  const history = useHistory();
+  const { step } = useParams<{ step: string }>();
+
+  const mailpoetAccountUrl = MailPoet.MailPoetComUrlFactory.getPurchasePlanUrl(
+    MailPoet.subscribersCount,
+    MailPoet.currentWpUserEmail,
+    'starter',
+    {
+      ref: 'plugin-wizard',
+      utm_source: 'plugin',
+      utm_medium: 'onboarding',
+      utm_campaign: 'purchase',
+    },
+  );
+
+  return (
+    <>
+      <Heading level={1}>
+        {MailPoet.I18n.t('welcomeWizardMSSFirstPartTitle')}
+      </Heading>
+
+      <div className="mailpoet-gap" />
+      <p>{MailPoet.I18n.t('welcomeWizardMSSFirstPartSubtitle')}</p>
+      <div className="mailpoet-gap" />
+
+      <div className="mailpoet-welcome-wizard-mss-list">
+        <List>
+          <li>{MailPoet.I18n.t('welcomeWizardMSSList1')}</li>
+          <li>{MailPoet.I18n.t('welcomeWizardMSSList2')}</li>
+          {subscribersCount < 1000 ? (
```

Comment:
`subscribersCount` is passed down too often while the value is a global, it can either come directly from window or using a utility function.

Reply:
Done, thanks